#3101 Redesign ETO documents

drakkein opened the pull request.

Moejoe90 commented on packages/web/app/modules/eto-documents/sagas.ts:
```diff
@@ -198,6 +198,19 @@ function* uploadEtoFileEffect(
+  { apiEtoFileService, notificationCenter }: TGlobalDependencies,
+  documentType: EEtoDocumentType,
+): Iterator<any> {
+  const matchingDocument = yield getDocumentOfType(documentType);
```
and what happens if there is no `matchingDocument` 
We sholdn't show 
```
  notificationCenter.info(createMessage(EtoDocumentsMessage.ETO_DOCUMENTS_FILE_REMOVED));
``` 
if matchingDocument wasn't there 

also report this case to sentry becuase something is really wrong

drakkein commented on packages/web/app/components/shared/Document.tsx:
```diff
@@ -69,106 +58,74 @@ export const Document: React.FunctionComponent<IDocumentProps> = ({
+        onClick={downloadAction}
+        type="button"
+        svgIcon={download}
+        alt={<FormattedMessage id="button-icon.down" />}
```
It is hidden behind loading screen, so it should not be possible for user to click on it, anyway I will disable it for better security

drakkein commented on packages/web/app/modules/eto-documents/sagas.ts:
```diff
@@ -226,12 +239,43 @@ function* uploadEtoFile(
       const uploadResult = Object.values(eto.documents).find(d => d.documentType === documentType)!;
-      yield put(actions.etoFlow.signInvestmentAgreement(eto, uploadResult.ipfsHash));
+
+      // If user does not sign transaction uploadResult is undefined
```
Basically if user close permission check modal/metamask signer during upload.

drakkein commented:
Fixed issue with uploading PDF's.
Fixed styling for templates box.

Downloading is related to CORS, should work normally on '.io'. Codechecks has some headers missing.
